techlibs/quicklogic/qlf_k6n10_brams.txt

```diff
@@ -0,0 +1,139 @@
+bram $__QLF_RAM16K_M0
+  init 1
+  abits 8
+  dbits 32
+  groups 2
+  ports  1  1
+  wrmode 0 1
+  enable 1 1
+  transp 0  0
+  clocks 2  3
+  clkpol 1  1
+endbram
+
+bram $__QLF_RAM16K_M1
+  init 1
+  abits  9
+  dbits  32
+  groups 2
+  ports  1 1
+  wrmode 0 1
+  enable 1 1
+  transp 0 0
+  clocks 2 3
+  clkpol 1 1
+endbram
+
+bram $__QLF_RAM16K_M2
+  init 1
+  abits  10
+  dbits  32
+  groups 2
+  ports  1 1
+  wrmode 0 1
+  enable 1 1
+  transp 0 0
+  clocks 2 3
+  clkpol 1 1
+endbram
+
+bram $__QLF_RAM16K_M3
+  init 1
+  abits 11
+  dbits  32
+  groups 2
+  ports  1 1
+  wrmode 0 1
+  enable 1 1
+  transp 0 0
+  clocks 2 3
+  clkpol 1 1
+endbram
+
+
+# The syn_* attributes are described in:
+# https://www.latticesemi.com/-/media/LatticeSemi/Documents/Tutorials/AK/LatticeDiamondTutorial311.ashx
```

Comment:
Since both read and write clock are driven by same clock network in k6n10 it should be same clock 
for "clocks" construct.

Reply:
Agreed, both read&write clocks are now driven by the same clock now.
I left both port in the bram module definition ( in cells_sim ) as I believe this should represent our bram cell definition.